### ISSUE TITLE
Using simclr, we completely do not need to label image data for pre-training. Am I right?

### ISSUE THREAD

**Issue by guotong1988** [google-research/simclr#48]
Or we need to label smaller image dataset?

Thank you very much!

**Comment by chentingpc** [comment]
no label for pretraining

**Comment by guotong1988** [comment]
![image](https://user-images.githubusercontent.com/4702353/86078727-4fe3b900-bac1-11ea-95f8-321375cb9f5c.png)

How could we know dog and chair should repel?
